bin/reth/src/commands/nodeset.rs

```diff
@@ -0,0 +1,79 @@
+//! Subcommand of P2P Debugging tool,
+
+use chrono::{DateTime, Utc};
+use clap::{Parser, Subcommand};
+use discv5::enr::Enr;
+use reth_fs_util as fs;
+use secp256k1::SecretKey;
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, io::Read, path::PathBuf};
+
+/// The arguments for the `reth p2p nodeset` command
+#[derive(Parser, Debug)]
+pub struct Command {
+    #[clap(subcommand)]
+    subcommand: Subcommands,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct Record {
+    seq: u64,
+    record: Enr<SecretKey>,
+    score: i64,
+    firstResponse: DateTime<Utc>,
+    lastResponse: DateTime<Utc>,
+    lastCheck: DateTime<Utc>,
+}
```

Comment:
we don't use any of the fields in this type except `record`, so let's delete this type

please open an issue for writing data for the other fields to the `known-peers.json` file, and we can discuss whether we should do that in the issue

Reply:
These fields will be used when implement `nodeset filter`.